test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -751,8 +751,148 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 		})
 	})
 
+	Context("Runc cpus isolation", func() {
+		var guaranteedPod, secondPod *corev1.Pod
+
+		AfterEach(func() {
+			cmd := []string{"rm", "-f", "/rootfs/var/roothome/create"}
+			nodes.ExecCommand(ctx, workerRTNode, cmd)
+			deletePod(ctx, guaranteedPod)
+			deletePod(ctx, secondPod)
+		})
+
+		It("[test_id: 74461] Verify that runc excludes the cpus used by guaranteed pod", func() {
+			guaranteedPod = getGuaranteedPod(ctx, workerRTNode)
+			secondPod = getNonGuaranteedPod(ctx, workerRTNode)
```

Comment:
can we give it more meaningful name like bestEffortPod or so?

Reply:
Updated with better name.